src/app/models/evt-models.ts

```diff
@@ -127,6 +127,23 @@ export interface WitnessGroup {
     groupId: string;
 }
 
+export type ApparatusEntriesList = Map<ApparatusEntry>;
+
+export interface ApparatusEntry extends GenericElementData {
+    id: string;
+    content: ApparatusReading[];
+    note: NoteData;
```

Comment:
Does TEI allow to have multiple `<note>`s within a single `<app>`? If so, we should maybe anticipate this particular encoding and handle `note` as an array of `NoteData`

Reply:
Done